workers/check_for_new_commits_on_github_worker.rb

```diff
@@ -68,4 +68,4 @@ def work
       end
     end
   end
-end
\ No newline at end of file
+end
```

Comment:
Gotta make sure we have a new line at the end of the file

Reply:
done.